syntax = "proto3";

package rgw.test.v1;

service RgwGrpcTestService {
    rpc Ping(PingRequest) returns (PingResponse);
}

message PingRequest {
    string message = 1;
}

message PingResponse {
    string message = 1;
}
